### PR TITLE
FE-184 legal address form

### PR DESCRIPTION
**Pull request by TiagoFuelber**
* Add Legal Address form section

https://developer.justifi.ai/tag/Business#operation/CreateBusiness

Links
-----

Developer considerations
--------------

- [ ] Increment the version number for relevant packages
  - [ ] This is a new MAJOR version (incompatible API changes were made)
  - [ ] This is a new MINOR version (functionality was added in a backwards compatible manner)
  - [ ] This is a new PATCH version (backwards compatible bug fixes)
- [ ] Update the CHANGELOG.md with a summary of the changes
  - If this is a MAJOR version, what are the breaking API changes? Was anything added to the API?
  - If this is a MINOR version, what feature was added? Why would consumers want this new version?
  - If this is a PATCH version, what was fixed?
- [ ] Update the appropriate documentation. Keep in mind that this mono repository has README files at the repository, project, and component levels. We want developers to be able to find what they need from the main README, so make sure to add links to component documentation if adding something new.

Steps for Testing/QA
--------------------

### CODE REVIEW

File: stencil-library/src/components/business-form/business-form-schema.ts [hunk 1]
```diff
@@ -135,21 +169,9 @@ const BusinessFormSchema = object({
...
-    dob_year: string().required('Enter representative birth year'),
-    identification_number: string().required('Enter representative identification number'),
-    address: object({
-      line1: string().required('Enter street address'),
-      city: string().required('Enter city'),
-      state: string().required('Select state'),
-      postal_code: string().required('Enter postal code'),
-    })
-  })
+  representative: representativeSchema.required('Enter representative information'),
```
Comment (jakemer) [inline stencil-library/src/components/business-form/business-form-schema.ts:172]: Nice - having these sections broken up into their own vars makes this a lot cleaner looking

File: stencil-library/src/components/business-form/business-form.tsx [hunk 2]
```diff
@@ -22,33 +22,39 @@ export class BusinessForm {
...
+    this.formController.validateAndSubmit(this.sendData);
   }
 
   render() {
     return (
       <Host exportparts="label,input,input-invalid">
         <h1>Business Information</h1>
-        <form onSubmit={(event) => this.validateAndSubmit(event)}>
-          <div class="row gy-3">
+        <form onSubmit={event => this.validateAndSubmit(event)} class="container">
```
Comment (jakemer) [inline stencil-library/src/components/business-form/business-form.tsx:37]: This change (`class="container"`) was in another PR and we decided not to include it. Should it be included here?

File: stencil-library/src/components/business-form/business-form-schema.ts [hunk 3]
```diff
@@ -135,21 +169,9 @@ const BusinessFormSchema = object({
...
+  owners: array().of(OwnerSchema).min(1, 'Enter at least 1 owners'),
+  legalAddress: legalAddressSchema.required('Enter legal address'),
```
Comment (jakemer) [inline stencil-library/src/components/business-form/business-form-schema.ts:174]: I believe this should be `legal_address` based on the API docs

File: stencil-library/src/components/legal-address-form/legal-address-form.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+export class LegalAddressForm {
+  @Prop() formController: FormController;
+  @Prop() legend: string;
+  @State() errors: any = {};
+  @State() defaultValues: any = {};
+  @State() legalAddress: any = {};
+
+  @Watch('legalAddress')
+  handleLegalAddressChange(newValue: any) {
+    this.formController.setValues({ legalAddress: newValue });
```
Comment (jakemer) [inline stencil-library/src/components/legal-address-form/legal-address-form.tsx:24]: I believe this should be `legal_address` based on the API docs

File: stencil-library/src/components/business-form/business-form.tsx (hunk 2 above)
Comment (TiagoFuelber) [reply stencil-library/src/components/business-form/business-form.tsx:37]: Oh, no, it was just bad merging on my end.

**Pull request merged**